src/Mod/OpenSCAD/OpenSCADCommands.py

```diff
@@ -387,7 +395,42 @@ def addelement(self):
 
         except OpenSCADUtils.OpenSCADError as e:
             FreeCAD.Console.PrintError(e.value)
-
+     
+    def refreshelement(self):
+        print('Refresh Element')
+        doc=FreeCAD.activeDocument()
+        if doc :
+           for obj in doc.Objects :
+               doc.removeObject(obj.Name)
+        self.addelement()
+
+    def loadelement(self):
+        print('Load Element')
+        print('Load Element')
+        filename, filter = QtGui.QFileDialog.getOpenFileName(parent=self.form, caption='Open file', dir='.', filter='OpenSCAD Files (*.scad)',selectedFilter='',option=0)
+
+        if filename:
+           print('filename :'+filename)
+           fp = open(filename,'r')
+           with fp:
```

Comment:
Use 'with ... as...' so fp is closed at the end of the with block
`with open(filename, 'r') as fp:`